
Waiting for JIT...
Traceback (most recent call last):
  File "/home/pedro/PINN/jaxpi/examples/geomII/main.py", line 39, in <module>
    app.run(main)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomII/main.py", line 31, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/home/pedro/PINN/jaxpi/examples/geomII/train.py", line 121, in train_and_evaluate
    log_dict = evaluator(state, batch, coords, u_ref, v_ref)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomII/models.py", line 353, in __call__
    self.log_errors(state.params, coords, u_ref, v_ref)
  File "/home/pedro/PINN/jaxpi/examples/geomII/models.py", line 335, in log_errors
    u_error, v_error = self.model.compute_l2_error(params, coords, u_ref, v_ref)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomII/models.py", line 246, in compute_l2_error
    u_error = jnp.linalg.norm(u_pred - u_test) / jnp.linalg.norm(u_test)
                              ~~~~~~~^~~~~~~~
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/array_methods.py", line 736, in op
    return getattr(self.aval, f"_{name}")(self, *args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/array_methods.py", line 264, in deferring_binary_op
    return binary_op(*args)
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/numpy/ufuncs.py", line 85, in <lambda>
    fn = lambda x1, x2, /: lax_fn(*promote_args(numpy_fn.__name__, x1, x2))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: sub got incompatible shapes for broadcasting: (4804,), (10024,).
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.